cmd/kube-scheduler/app/options/options_test.go

```diff
@@ -528,3 +532,19 @@ pluginConfig:
 		})
 	}
 }
+
+func stringPtr(s string) *string {
```

Comment:
https://github.com/kubernetes/utils/blob/master/pointer/pointer.go has all the functions bellow already defined.

Reply:
Oh great! Wasn't aware of that package, thanks